lib/portfinder.js

```diff
@@ -167,6 +167,27 @@ exports.getPort = function (options, callback) {
 };
 
 //
+// ### function getPortPromise (options)
+// #### @options {Object} Settings to use when finding the necessary port
+// Responds a promise to an unbound port on the current machine.
+//
+if (typeof Promise === 'function') {
```

Comment:
Lets export this api regardless of whether native promise support is missing. If Promise is not available, lets still export the method and `throw Error('Native promise support is not available in this version of node. Please install a polyfill and assign Promise to global.Promise before calling this method`

or something like that

tl;dr - i dont want to conditionally export a public api method without a nice warning that the runtime does not support it, otherwise the error would be `portfinder.getPortPromise is undefined/not a function` or whatever - u get the drift

we could include bluebird but i don't want to bloat the size of the lib either

Feel free to notify the user w/ a message u see fit, including relevant details. Maybe update the README.md to mention this too.

thanks again - this looks good otherwise